huxley/settings/registration.py

```diff
@@ -1,4 +0,0 @@
-# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
-# Use of this source code is governed by a BSD License (see LICENSE).
-
-REGISTRATION_OPEN = True
```

Comment:
Did you remove the references in `huxley/api/views/user.py` and in the tests in `huxley/api/tests/test_user.py` ?